### ISSUE TITLE
Missing module `chalk`

### ISSUE THREAD

**Issue by brandonparsons** [tildeio/ember-cli-fastboot#4]
```
$ ember fastboot
version: 0.2.0-beta.1
Built project successfully. Stored in "dist".
Cannot find module 'chalk'
Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/brandon/code/myapp/node_modules/ember-cli-fastboot/lib/models/server.js:2:13)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

Nothing in there except a completely bare ember app, I followed the directions as per the README.

**Comment by tomdale** [comment]
@brandonparsons Really sorry about that, I was doing some last-minute hacking at EmberConf and inadvertently put some dependencies in `devDependencies` instead. This should be fixed in 0.0.4, let me know if you're still having issues! Sorry again.

**Comment by brandonparsons** [comment]
Haha no problem :) Just wanted to play around with it!
